vms/volunteer/models.py

```diff
@@ -87,7 +87,7 @@ class Volunteer(models.Model):
         blank=True,
         validators=[
             RegexValidator(
-                r'^[(A-Z)|(a-z)|(\s)|(\.)|(\-)|(?)|(=)|(#)|(:)|(/)|(_)|(&)]+$',
+                r'^[(A-Z)|(a-z)|(0-9)|(\s)|(\.)|(\-)|(?)|(=)|(#)|(:)|(/)|(_)|(&)]+$',

```

Comment:
This will allow the user to just use numbers which is not correct, you have digits ```ored``` now. As an example this now accepts ```123.123```. I just observed these regex(es) are quite wrong. Can you modify it to a valid regex for a URL?